modules/sdk-coin-avaxp/src/lib/transactionBuilder.ts

```diff
@@ -108,7 +108,7 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
    * @param {string | stirng[]} senderPubKey
    */
   fromPubKey(senderPubKey: string | string[]): this {
-    const pubKeys = senderPubKey instanceof Array ? senderPubKey : [senderPubKey];
+    const pubKeys = senderPubKey instanceof Array ? senderPubKey : senderPubKey.split('~');
```

Comment:
readability-nit: add const for ADDRESS_SEPARATOR

Reply:
Sender Pub key order is relevante. The multi address convention sort them. So we should avoid use them here.